src/index.js

```diff
@@ -40,19 +39,19 @@ function rawr(event) {
     document.querySelector("head").innerHTML = "";
     document.querySelector("div#root").remove()
     setTimeout(() => {
-      document.querySelector("img#whOwOps").setAttribute("src", `/assets/whOwOps/whOwOps${++i}.png`);
+      document.querySelector("img#whOwOps").setAttribute("src", document.querySelector("img#whOwOps").getAttribute("src").replace(/(\D+)(\d+)(\.png$)/, (match, prefix, num, suffix) => `${prefix}${parseInt(num, 10) + 1}${suffix}`));
       setTimeout(() => {
-        document.querySelector("img#whOwOps").setAttribute("src", `/assets/whOwOps/whOwOps${++i}.png`);
+        document.querySelector("img#whOwOps").setAttribute("src", document.querySelector("img#whOwOps").getAttribute("src").replace(/(\D+)(\d+)(\.png$)/, (match, prefix, num, suffix) => `${prefix}${parseInt(num, 10) + 1}${suffix}`));
         setTimeout(() => {
-          document.querySelector("img#whOwOps").setAttribute("src", `/assets/whOwOps/whOwOps${++i}.png`);
+          document.querySelector("img#whOwOps").setAttribute("src", document.querySelector("img#whOwOps").getAttribute("src").replace(/(\D+)(\d+)(\.png$)/, (match, prefix, num, suffix) => `${prefix}${parseInt(num, 10) + 1}${suffix}`));
           setTimeout(() => {
-            document.querySelector("img#whOwOps").setAttribute("src", `/assets/whOwOps/whOwOps${++i}.png`);
+            document.querySelector("img#whOwOps").setAttribute("src", document.querySelector("img#whOwOps").getAttribute("src").replace(/(\D+)(\d+)(\.png$)/, (match, prefix, num, suffix) => `${prefix}${parseInt(num, 10) + 1}${suffix}`));
             setTimeout(() => {
-              document.querySelector("img#whOwOps").setAttribute("src", `/assets/whOwOps/whOwOps${++i}.png`);
+              document.querySelector("img#whOwOps").setAttribute("src", document.querySelector("img#whOwOps").getAttribute("src").replace(/(\D+)(\d+)(\.png$)/, (match, prefix, num, suffix) => `${prefix}${parseInt(num, 10) + 1}${suffix}`));
               setTimeout(() => {
-                document.querySelector("img#whOwOps").setAttribute("src", `/assets/whOwOps/whOwOps${++i}.png`);
+                document.querySelector("img#whOwOps").setAttribute("src", document.querySelector("img#whOwOps").getAttribute("src").replace(/(\D+)(\d+)(\.png$)/, (match, prefix, num, suffix) => `${prefix}${parseInt(num, 10) + 1}${suffix}`));
                 setTimeout(() => {
-                  document.querySelector("img#whOwOps").setAttribute("src", `/assets/whOwOps/whOwOps${++i}.png`);
+                  document.querySelector("img#whOwOps").setAttribute("src", document.querySelector("img#whOwOps").getAttribute("src").replace(/(\D+)(\d+)(\.png$)/, (match, prefix, num, suffix) => `${prefix}${parseInt(num, 10) + 1}${suffix}`));
```

Comment:
This is a good suggestion, and I did consider this while rewriting code, as before, the entire file path and numbers were hardcoded into the logic. However, I think this suggestion is actually more complicated than the previous one, and we're sacrificing readability for the Do Not Repeat yourself principle. Replacing 7 identical one-liners with three function, spanning more lines of code than the original, is not the best thing, IMO. And, now that the lines are identical, instead of having different number value hardcoded in them, it's not that hard to just do Find and Replace on all of them. So, for me, this one-liner version seems better.
I think it'd be a good idea to get an outsider opinion, though. I will ask @HostyIsDead for an opinion on this.

Reply:
@coderabbitai After a small conversation with another developer, @SlovakTastic, we have come to the conclusion, that chaining with functions this way, in this context, is unnecessary, and overly complicated.
We have come to a similar conclusion with @Hoshty.
The way the code is structured currently is just 7 identical one-liners, and making it 3 functions (which are, when added together, taking up way more space than the original code) is not the best solution at a small scale like this.